lib/frontend/CMakeLists.txt

```diff
@@ -0,0 +1,23 @@
+set(LLVM_LINK_COMPONENTS
+  Option
+  Support
+  )
+
+
+llvm_add_library(flangFrontend
+  TextDiagnosticPrinter.cpp
+
+  LINK_LIBS
+  clangBasic
+  clangDriver
+  )
+
+include_directories(${LLVM_DIR} ${LLVM_DIR}/../..)
```

Comment:
We need to include the llvm lib directory. I have replaced that for link_directories(${LLVM_LIBRARY_DIR}).
But libclangBasic and lib clang driver will only be found if the CMake can point to its binary otherwise we fail to build.